lesson-9-maximum-slice-problem/maxslicesum.py

```diff
@@ -0,0 +1,7 @@
+def solution(A):
+    max_ending = max_slice = -1000000
```

Comment:
that's the minimum range given in the problem. assigning it increases space consumption, and it's only an initial value